x/auth/keeper/keeper.go

```diff
@@ -243,7 +243,10 @@ func (ak AccountKeeper) GetModuleAccountAndPermissions(ctx context.Context, modu
 	}
 
 	// create a new module account
-	macc := types.NewEmptyModuleAccount(moduleName, perms...)
+	macc, err := types.NewEmptyModuleAccount(ak.addressCodec, moduleName, perms...)
```

Comment:
i think we should leave some of the globals around here. Since we are thinking of moving away from the auth keeper i dont think we should do a bunch of api breaks then in the future remove it.